### ISSUE TITLE
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2026' in position 110: ordinal not in range(128)

### ISSUE THREAD

**Issue by strugee** [dper/pumptweet#29]
For some reason #14 seems to have cropped up again:

```
Traceback (most recent call last):
  File "./pt.py", line 21, in <module>
    pumptweet.pull_and_push(halt_on_error=halt_on_error)
  File "/home/alex/pumptweet_community/pumptweet/PumpTweet.py", line 189, in pull_and_push
    self.post_tweets(tweets)
  File "/home/alex/pumptweet_community/pumptweet/PumpTweet.py", line 160, in post_tweets
    self.post_tweet(tweet)
  File "/home/alex/pumptweet_community/pumptweet/PumpTweet.py", line 149, in post_tweet
    print tweet
UnicodeEncodeError: 'ascii' codec can't encode character u'\u2026' in position 110: ordinal not in range(128)
```

The offending note appears to be https://pumpit.info/pumpio/note/_FM0E8BhTwOwz6ThhWbhqQ. Note that that note was edited after the fact, to add a title.

**Comment by dper** [comment]
The `u'\u2026'` thing appears to be a horizontal ellipsis.  That is peculiar because I don't see one in your note.  So perhaps something strange is going on.

Right now it seems microca.st is down, so I can't easily debug.  I'll take a closer look when it's back up.  :-/

**Comment by strugee** [comment]
@dper thanks, as always.

**Comment by strugee** [comment]
I should also point out that I'm just assuming that's the problematic note based on the fact that it's the note directly after the most recently-syndicated note. But I could be wrong.

**Comment by dper** [comment]
I tried replicating this https://microca.st/dper/note/CQkBbFmwSee_i1g63oCcyA using a message that is letter-for-letter the same as the note mentioned by @strugee.

And then because it could have been a Pumpa (me) vs. web client problem (strugee) behavior difference, I tried a multi-line web post https://microca.st/dper/note/sQtBTXmCTy2F4ThHA2Wfpw (although this one was not identical).

Both of those work as normal, at least for me.  Thoughts?
